pages/docs/configuration/configuration-acceleration.md

```diff
@@ -26,17 +27,29 @@ All data communicated within a coupling scheme needs to be configured through `e
+**Example:** For fluid-structure interaction ignoring the velocities, if we first execute the fluid solver with given interface displacements followed by the structure solver taking forces from the fluid solver and computing new interface displacements, (only) the displacements are accelerated in case of serial coupling. For parallel coupling, both displacements and forces are accelerated.
 
-Next, we have to configure based on which data the acceleration computes, i.e. how the coefficients in the linear combinations get computed. These data fields are defined within the acceleration as `data` tags (such as `Displacements` in the code example above). Let's call these data fields **primary data**. (Just for completeness: All coupling data that gets post-processed and that is not primary data, is called "secondary data".)
+Different acceleration schemes compute these coefficients in different ways.
+They are generally based on the values of previous iterations.
+A notable exception is the constant under-relaxation, which uses fixed coefficients.
 
-* For **serial coupling**, you can only configure one primary data field, which should correspond to a coupling data field that is exchanged from the `second` to the `first` participant. In the FSI example, the `Displacements`.
-* For **parallel coupling**, an arbitrary number of primary data can be configured. For numerical performance reasons, you should define at least one coupling data field of each direction (one from `second` to `first`, one from `first` to `second`). In the FSI example, configure `Displacements` and `Forces`.
+Such value-dependent acceleration schemes need to select which data to compute these coefficients from by listing them as `data` tags inside the `acceleration` tag.
+We call data which influences the coefficients **primary data** and data which is accelerated without influencing the coefficients **secondary data**.
+The former is explicitly listed in the `acceleration` using `data` tags, while the latter is implied by the primary data and the configured coupling scheme.
+Which data may be configured depends on the coupling scheme:
+
+* For **serial coupling**, you can only configure primary data from coupling data which is exchanged from the `second` to the `first` participant. In the FSI example, the `Displacements`.
+* For **parallel coupling**, all coupling data is available as primary data. For numerical performance reasons, you should define at least one coupling data field of each direction (one from `second` to `first`, one from `first` to `second`). In the FSI example, configure `Displacements` and `Forces`.
+
+In the code example above, `Displacements` is primary data and `Velocities` is secondary data. `Forces` is not accelerated as the case uses a serial coupling scheme.
+Changing the `serial-implicit` to a `parallel-implicit` scheme would turn `Forces` into secondary data.
```

Comment:
I feel that such an overview would help get a quick idea:

```suggestion
In the code example above, `Displacements` is primary data and `Velocities` is secondary data. `Forces` is not accelerated as the case uses a serial coupling scheme:

| Category            | Data fields |
| ---                 | ---         |
| Exchanged           | `Displacements`, `Velocities`, `Forces` |
| Accelerated         | `Displacements`, `Velocities` |
| Define coefficients | `Displacements` |
| Not accelerated     | `Forces` |

Changing the `serial-implicit` to a `parallel-implicit` scheme would turn `Forces` into secondary data.
```

Reply:
Feels like overkill.

If we add such a table, then I would suggest doing it as a summary adding columns for serial and parallel cases.


| Category            | Serial Coupling | Parallel Coupling |
| ---                 | ---         | --- |
| Exchanged           | `Displacements`, `Velocities`, `Forces` | `Displacements`, `Velocities`, `Forces` |
| Accelerated         | `Displacements`, `Velocities` | `Displacements`, `Velocities`, `Forces` |
| Define coefficients | `Displacements` | `Displacements`, `Forces` |
| Not accelerated     | `Forces` | - |

Reply:
Such a table would help me, but if you think this is an overkill, I guess we are fine if people don't understand as easily as we could enable them to. :upside_down_face: 

I like the distinction with the serial and parallel coupling.

Reply:
If it helps you, then we can easily add it. @uekerman any thoughts / improvements?

The categories are inconsistent though. We talk about primary and secondary data, and then use "define coefficients" and "accelerated".

Reply:
As I wrote:

> I think that the terminology "primary/secondary" is not so important / does not help here, but it could also go into a third, "comments" column.

Reply:
```suggestion
In the code example above, `Displacements` is primary data and `Velocities` is secondary data. `Forces` is not accelerated as the case uses a serial coupling scheme:

| Category | Serial Coupling | Parallel Coupling |
| ---  | ---  | --- |
| Exchanged | `Displacements`, `Velocities`, `Forces` | `Displacements`, `Velocities`, `Forces` |
| Accelerated | `Displacements`, `Velocities` | `Displacements`, `Velocities`, `Forces` |
| Define coefficients (primary) | `Displacements` | `Displacements`, `Forces` |
| Exchanged but not accelerated (secondary) | `Forces` | - |

Changing the `serial-implicit` to a `parallel-implicit` scheme would turn `Forces` into secondary data.
```

Reply:
But we introduce these terms for the sole reason to explain what is happening in preCICE. Using the suggested categories does question the value of the entire section.

That said, this may as well be a symptom, that "primary/secondary" is a useless classification. Maybe explaining what is accelerated and what is used to compute the coefficients is way easier to follow without attempting to tie them to "primary/secondary".

Reply:
I don't challenge the terms: I understand there are good (literature) reasons that they are named like this.

I am just trying to help new readers get quicker to the point and understand what is going on and why. In my latest suggestion, I have added both the description and the names in the first column.

Reply:
I also think that such a table is overkill. I see all this information here merely as docs for devs, especially primary vs. secondary. For users not interested in the math, we could also just write "trust that preCICE does the right thing".

Reply:
ok, go ahead.